packages/preview-middleware-client/src/cpe/changes/validator.ts

```diff
@@ -0,0 +1,30 @@
+import ResourceBundle from 'sap/base/i18n/ResourceBundle';
+
+/**
+ * Function to validate if a given value is a valid binding model.
+ *
+ * @param value value to be checked.
+ */
+export function vaildateBindingModel(value: string): void {
+    const bindingValue = value.replace(/{|}/gi, '');
+    const bindingParts = bindingValue.trim().split('>').filter(el => el);
```

Comment:
Are you sure about this one? What is you filter condition?

Reply:
When you split and the value ends with the split char the return array contains one additional empty string element.